cfme/fixtures/v2v.py

```diff
@@ -41,6 +37,269 @@ def conversion_tags(appliance, host_creds):
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
         display_name='VDDK')
+    host_creds.remove_tags(host_creds.get_tags())
     host_creds.add_tags(tags=(tag1, tag2))
     yield
     host_creds.remove_tags(tags=(tag1, tag2))
+
+
+def get_vm(request, appliance, second_provider, template, datastore='nfs'):
+    """Fixture to provide vm object"""
+    source_datastores_list = second_provider.data.get('datastores')
+    source_datastore = [d.name for d in source_datastores_list if d.type == datastore][0]
+    collection = second_provider.appliance.provider_based_collection(second_provider)
+    vm_obj = collection.instantiate(random_vm_name('v2v'),
+                                    second_provider,
+                                    template_name=template(second_provider)['name'])
+
+    request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
+    vm_obj.create_on_provider(timeout=2400, find_in_cfme=True, allow_skip="default",
+         datastore=source_datastore)
+    return vm_obj
+
+
+def _form_data_cluster_mapping(second_provider, provider):
+    # since we have only one cluster on providers
+    source_cluster = second_provider.data.get('clusters')[0]
```

Comment:
This pattern is raising `TypeError` in current implementations because a `.get()` call can return NoneType, which has no `__getitem__`

Reply:
Made specific fixes in #7650